src/objects/zcl_abapgit_objects_program.clas.abap

```diff
@@ -782,10 +782,10 @@ CLASS ZCL_ABAPGIT_OBJECTS_PROGRAM IMPLEMENTATION.
                    <ls_field_int>   LIKE LINE OF lt_fieldlist_int.
 
     "#2746: relevant flag values (taken from include MSEUSBIT)
-    CONSTANTS:    lc_flg1ddf TYPE x VALUE '20',
-                  lc_flg3fku TYPE x VALUE '08',
-                  lc_flg3for TYPE x VALUE '04',
-                  lc_flg3fdu TYPE x VALUE '02'.
+    CONSTANTS: lc_flg1ddf TYPE x VALUE '20',
```

Comment:
no need to do any changes in this file/no changes to this file, in this case it just makes the PR more difficult to review

Reply:
You're right, it was changed in a previous commit, I restored it